README.md

```diff
@@ -93,12 +93,19 @@ To run stress testing against the Salesforce instance, refer to the documentatio
 To get started working with our Phrase translations, you will need to:
 
 1. Install the Phrase CLI with `brew tap phrase/brewed && brew install phrase`
-1. [Create an access token for Phrase](https://app.phrase.com/settings/oauth_access_tokens). Save it for future use in e.g. LastPass
+1. [Create an access token for Phrase](https://app.phrase.com/settings/oauth_access_tokens). Save it for future use in Lastpass and as a local env var.
 
-**To upload new strings or translations to Phrase**
+- Each time you add new strings to locale-en.json, push to Phrase to keep it up-to-date
```

Comment:
I just realized these aren't really sub-bullets of the steps, can we make this section look something like:
```
## Translation processes
**When strings are added to or deleted from locale-en.json**
run phrasePush after merging your PR // contingent on if you can always run cleanup from phrasePush script.

**When you need to directly update non-english phrases**
   1. Update the phrases in Phrase
   2. run phrasePull 
```

Then you can delete the  **To upload new strings to Phrase** and  **To download new translations from Phrase** sections since those commands are covered above

Reply:
updated! let me know if changes make sense. We have to pull for the case that we have new translations from our partners, and also for the case where we rename keys and have to update the translations in phrase